stratosphere/ams_mitm/source/set_mitm/setsys_mitm_service.cpp

```diff
@@ -87,6 +88,87 @@ namespace ams::mitm::settings {
             return ResultSuccess();
         }
 
+        const char *g_external_bluetooth_db_location = "@Sdcard:/atmosphere/bluetooth_devices.db";
```

Comment:
If this is a true constant, please put it in an anonymous namespace at top of file (inside the file's normal namespace), and also please mark it constexpr.

`constexpr const char ExternalBluetoothDatabasePath[] = "@Sdcard:/atmosphere/bluetooth_devices.db";`

Reply:
Wanted to ask about this one. Is it fine to write the whole path out as a constant like this? I saw there was `ams::fs::impl::SdCardFileSystemMountName` for the mount name and abunch of helper functions in `amsmitm_fs_utils.hpp`

Also, are you fine with storing the db in the `/atmosphere` root directory or should it go in a subdirectory somewhere?